__tests__/main.test.ts

```diff
@@ -29,11 +29,18 @@ describe('Run Github Action', () => {
   })
   describe('Handle input parameters', () => {
     test('Github Action called with dummy parameter fails with core.setFailed', async () => {
+      const originalResolveConfig = resolveConfigModule.resolveConfig
+      jest.spyOn(resolveConfigModule, 'resolveConfig').mockImplementation(async reporter => ({
+        ...(await originalResolveConfig(reporter)),
+        failOnCriticalErrors: true,
+      }))
+
       const setFailedMock = jest.spyOn(core, 'setFailed')
-      resolveConfigModule.DEFAULT_CONFIG.failOnCriticalErrors = true
```

Comment:
Why not use `synthetics.DEFAULT_CONFIG`?

Reply:
It would have worked, but since `synthetics` comes from an external dependency (datadog-ci), now for some reason I felt wrong doing this... so I preferred the `spyOn` way 🤷

Reply:
This line will be removed anyway once we add the `INPUT_FAIL_ON_CRITICAL_ERRORS`, so actually let me do what you said @etnbrd